components/movementsensor/gpsrtki2c/gps_rtk_i2c.go

```diff
@@ -0,0 +1,613 @@
+// Package gpsrtki2c implements a gps using serial connection
```

Comment:
[nit] I think we like/prefer the filenames that have the package meat to match the package name, but I'm not too bothered by that. It's just easier for someone looking through our code base to search for using an exact matching package name.